uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/task-list.js

```diff
@@ -1,376 +1,383 @@
-define([
```

Comment:
Did you simply run semistandard * --fix ?
Any manual changes?

Reply:
Well, its because I ran `run semistandard * --fix`. With some manual changes.

Reply:
Could you point out these manual changes? What were they: indentation, names, logic?

Reply:
Line no: 4, 11, 268,  are changes from `==` to `===`. And line no 357-366 is small logic separation as there was mixing of many `&&` and `||`.